public/components/agents/vuls/inventory.tsx

```diff
@@ -84,18 +178,113 @@ export class Inventory extends Component {
     </EuiPage>;
   }
 
+  beautifyDate(date: string) {
+    return ['', '1970-01-01T00:00:00Z'].includes(date) ? '-' : formatUIDate(date);
```

Comment:
**nitpick**: It could be interesting to add a comment to express the reason for this logic.